walkthrough/grapher_cookiecutter/{{cookiecutter.namespace}}/{{cookiecutter.version}}/{{cookiecutter.short_name}}.py

```diff
@@ -28,7 +24,9 @@ def run(dest_dir: str) -> None:
     # Save outputs.
     #
     # Create a new grapher dataset with the same metadata as the garden dataset.
-    ds_grapher = create_dataset(dest_dir, tables=[tb], default_metadata=ds_garden.metadata)
+    ds_grapher = create_dataset(
+        dest_dir, tables=[tb], default_metadata=ds_garden.metadata, check_variables_metadata=True
+    )
```

Comment:
See my comment above about including grapher checks as part of `create_dataset`, if possible.